.github/workflows/reusable_build.yml

```diff
@@ -447,6 +451,33 @@ jobs:
         run: |
           echo CONFIG_KERNEL_WERROR=y >> .config
 
+      - name: Configure various options which are used on buildbots
+        shell: su buildbot -c "sh -e {0}"
+        working-directory: openwrt
+        run: |
+          echo CONFIG_JSON_CYCLONEDX_SBOM=y >> .config
+          echo CONFIG_COLLECT_KERNEL_DEBUG=y >> .config
+          echo CONFIG_REPRODUCIBLE_DEBUG_INFO=y >> .config
+          echo CONFIG_KERNEL_BUILD_USER="builder" >> .config
+          echo CONFIG_KERNEL_BUILD_DOMAIN="buildhost" >> .config
+
+      - name: Configure BPF options as used on buildbots
+        shell: su buildbot -c "sh -e {0}"
+        if: inputs.build_bpf_toolchain == true
+        working-directory: openwrt
+        run: |
+          echo CONFIG_DEVEL=y >> .config
+          echo CONFIG_SDK_LLVM_BPF=y >> .config
```

Comment:
I'm unsure where to put this config option, had it initially in next "Configure SDK and IB options used on buildbots" step, will recheck later.